hg-fast-export.py

```diff
@@ -122,6 +127,27 @@ def get_author(logmessage,committer,authors):
       return r
   return committer
 
+def generate_git_lfs_pointer(filename):
+  size = 0
+  hasher = hashlib.sha256()
+  with open(filename, "rb") as bfile:
+    buf = bfile.read(1024*1024*1024)
+    while buf:
+      hasher.update(buf)
+      size = size + len(buf)
+      buf = bfile.read(1024*1024*1024)
+  digest = hasher.hexdigest()
+  try:
+    os.makedirs("lfs/objects/%s/%s" % (digest[0:2],digest[2:4]))
+  except:
+    pass
+
```

Comment:
To me this looks like you look up the locally cached copy of the file and hash it. What happens if the file isn't in the user's cache? Isn't the whole point of the extension that you should not have to download all historic versions of the file? According to https://www.mercurial-scm.org/wiki/LargefilesExtension the standin in .hglf/ contains the sha1 of the file contents, why do you then have to hash the file and then copy the file into the cache on the git side?

To me a much cleaner way would be to for each standin on the hg side produce a pointer file on the git side, this should be possible to do without actually touching the large file, or?